lib/pxenforcer.js

```diff
@@ -285,8 +292,11 @@ class PxEnforcer {
         }
 
         // If verified, pass the request here
-        const shouldBypassMonitor = this._config.BYPASS_MONITOR_HEADER && req.headers[this._config.BYPASS_MONITOR_HEADER] === '1';
-        if (verified || ctx.monitoredRoute || (this._config.MODULE_MODE === this._config.MONITOR_MODE.MONITOR && !shouldBypassMonitor)) {
+        if (
+            verified ||
+            ctx.monitoredRoute ||
```

Comment:
Its the same check as in:
https://github.com/PerimeterX/perimeterx-node-core/pull/160/files#diff-2ac432be08cf7ea8a59f1141e164fc0799877adc4d26ffd84fcba0533f16a774R40

Please make a method to make sure that we will never forget to change both